### ISSUE TITLE
Stamina Movement Speed Change Not Being Incurred with 1 Stamina

### ISSUE THREAD

**Issue by greeduomacro** [ZaneDubya/UltimaXNA#394]
Hi Zane, love your work on this project and thank you so so so much for developing this client! 

As for the issue, specifically, OSI made stamina movement checks tied to the client and not the server as annoying as that is. Anyway, this is a pretty big problem for balance reasons because if a player connects to a shard with UltimaXNA for example they can completely bypass these checks. In particular in this context lies the issue:
On the original client at 1 stamina you can only walk and at 0 stamina you can't move at all.
On UltimaXNA at 1 stamina you can run or walk and at 0 stamina you can't move at all.
In other words the problem is for the speed of movement at 1 stamina..
As a side note the enhanced client has this same issue on all private shards heh.

Come to think of it, there are other variables possibly as well. See this post for more details:
http://uorevealed.com/index.php?threads/enhanced-client-unbalanced.351/#post-2306

**Comment by ZaneDubya** [comment]
@greeduomacro Just to confirm, if I add a check that will force walking when stamina == 1 and no movement when stamina == 0, this will fix the issue?

Thanks!

**Comment by greeduomacro** [comment]
Yes sir you got it. ^__^

**Comment by ZaneDubya** [comment]
Fixed in #435 - can't move at 0 stam, can only walk at 1.
